package.json

```diff
@@ -13,7 +13,7 @@
         "watch": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --watch --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
         "hot": "cross-env NODE_ENV=development webpack-dev-server --inline --hot --config=node_modules/laravel-mix/setup/webpack.config.js",
         "prod": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
-        "test": "jest --verbose --coverage --coverageDirectory=./report/jest/",
+        "test": "snyk test && jest --verbose --coverage --coverageDirectory=./report/jest/",
```

Comment:
Can we have a separate command for this, or do you need it to run `test` as part of the pipeline? Locally it would be nice to not have to run a security tool when running our test suite.

Reply:
Sure, no problem. The `snyk wizard` CLI command added it like that.